package.json

```diff
@@ -33,8 +33,8 @@
     "ink-docstrap": "^1.3.0",
     "istanbul-instrumenter-loader": "^2.0.0",
     "jasmine-sinon": "^0.3.2",
-    "jquery": "3.5.1",
-    "jquery-migrate": "3.3.1",
+    "jquery": "3.6.0",
+    "jquery-migrate": "3.3.2",
```

Comment:
Since we're modifying `package.json` do we also need to change the ventana version listed in this file? I know https://github.com/sugarcrm/ventana/pull/86/files is already bumping it to 1.1.12, I'm guessing that bumping it once per release is sufficient, kind of like what we do with API versioning?

Reply:
i think it would be best to bump the version at the same commit we publish to npm at. It's easier to keep track of what each npm release contains.

If i bump ventana here, and then another commit goes in after and then we publish to npm, it's kinda unclear what that package contains when going through commit history